index.html

```diff
@@ -734,12 +734,12 @@ <h5>Modify image and sound speed</h5>
     <div class="hiding">
       <h5>Fade both video and audio streams</h5>
       <p><code>ffmpeg -i <em>input_file</em> -filter:v "fade=in:st=0:d=1, fade=out:st=59:d=1" -filter:a "afade=in:st=0:d=1, afade=out:st=59:d=1" -c:v libx264 -c:a aac <em>output_file</em></code></p>
-      <p>This command fades in and out the first and last 30 video frames and 44100 audio samples on a video</p>
+      <p>This command fades in and out the first and last second (and 44100 audio samples) on a video.</p>
```

Comment:
Wondering where the `44100 audio samples` came from? What if the input isn't 44.1 kHz?